ddtrace/ext/git.py

```diff
@@ -152,7 +152,8 @@ def extract_git_metadata(cwd=None):
         log.error("Git executable not found, cannot extract git metadata.")
     except ValueError as e:
         debug_mode = log.isEnabledFor(logging.DEBUG)
-        log.error("Error extracting git metadata: %s", str(e), exc_info=debug_mode)
+        stderr = str(e)
```

Comment:
?
```suggestion
        stderr = compat.ensure_text(stderr).strip()
```

Reply:
`ensure_text` only works for text, but e is of type `ValueError`. Casting a `ValueError` to string returns the error message passed to the exception when it was created. This works in python2 and 3. Note that the error message has already been passed through `ensure_text` earlier.

Reply:
```suggestion
        stderr = str(e) # Returns the message passed to the exception when it was created
```

Reply:
We can add a comment to explain this, feel free to commit the above suggestion if you think it's worth it.